### ISSUE TITLE
cmvision_3d on ros jade

### ISSUE THREAD

**Issue by toddwf** [OSUrobotics/cmvision_3d#3]
I got this running on ros jade with changes on 

```
def depth_callback(self, image)
```

diff --git a/scripts/color_controller.py b/scripts/color_controller.py
index 2054b5f..8594fbd 100755
--- a/scripts/color_controller.py
+++ b/scripts/color_controller.py
@@ -80,8 +80,9 @@ class color_controller():

```
    def depth_callback(self, image):
```
-               image_cv = self.bridge.imgmsg_to_cv(image, image.encoding)
-               image_cv2 = np.array(image_cv, dtype=np.float32)
-               image_cv = self.bridge.imgmsg_to_cv2(image, image.encoding)
-               image_cv2 = np.squeeze(np.array(image_cv, dtype=np.float32))
              self.depth_image = image_cv2

**Comment by toddwf** [comment]
pull request issued

**Comment by ahubers** [comment]
Closed via #4
